nevergrad/optimization/test_optimizerlib.py

```diff
@@ -196,6 +196,90 @@ def test_infnan(name: str) -> None:
         )
 
 
+def suggestable(name: str) -> bool:
+    # Some methods are not good with suggestions.
+    keywords = ["TBPSA", "BO", "EMNA", "EDA", "BO", "Stupid", "Pymoo"]
+    return not any(x in name for x in keywords)
+
+
+def suggestion_testing(
+    name: str,
+    instrumentation: tp.Union[ng.p.Array, ng.p.Instrumentation],
```

Comment:
```suggestion
    instrumentation: np.p.Array,
```
you only use Array, and the tests will fail with anything else anyway